src/api/instance.js

```diff
@@ -1,15 +1,38 @@
 import axios from 'axios';
+import jwt_decode from 'jwt-decode';
+import { useDispatch, useSelector } from 'react-redux';
 
 const instance = axios.create({
     baseURL: import.meta.env.VITE_BASE_URL,
     timeout: 5000,
 });
 
 instance.interceptors.request.use(
-    function (config) {
+    async (config) => {
+        let decodeToken = ''
+        if (localStorage.getItem('accessToken')) {
+            decodeToken =jwt_decode(localStorage.getItem('accessToken'));
+        }
+        // const decodeToken =jwt_decode(localStorage.getItem('accessToken'));
+        // console.log(import.meta.env.VITE_BASE_URL);
+        if (decodeToken.exp < new Date().getTime()/1000) {
+            const refreshTokens = await axios.post(`http://localhost:8080/api/refrehToken`, { withCredentials: true })
```

Comment:
change local8080 to import.meta.env.VITE_BASE_URL

Reply:
chỗ này tạm thời chưa xong